config/cypress.json

```diff
@@ -8,168 +8,161 @@
   "testFiles": "**/tests/**/*.cypress.spec.js?(x)",
   "waitForAnimations": false,
   "env": {
-    "vaTop5MobileViewports": [
+    "vaTopMobileViewports": [
       {
-        "name": "iPhone XS Max, iPhone XR, iPhone 11, iPhone 11 Pro Max",
+        "list": "VA Top Mobile Viewports",
```

Comment:
Does this `list` key correspond to which group of viewports this particular object belongs to?

Reply:
Yes, it does.